stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/handlers/ClipboardHandler.kt

```diff
@@ -38,6 +43,11 @@ public fun interface ClipboardHandler {
  * @param clipboardManager System service that allows for clipboard operations, such as putting
  * new data on the clipboard.
  */
+@Deprecated(
+    message = "Use `ClipboardHandlerImpl` from `ui.common.helper` instead.",
+    replaceWith = ReplaceWith("ClipboardHandlerImpl", "io.getstream.chat.android.ui.common.helper"),
+    level = DeprecationLevel.WARNING,
+)
```

Comment:
Shouldn't we use the implementation of `io.getstream.chat.android.ui.common.helper.ClipboardHandlerImpl` here?

Reply:
That interface and class are not used anywhere in the project and are duplicates of those in `io.getstream.chat.android.ui.common.helper`
That's why I didn't remove it, just marked it as deprecated.